Woof/Woof/View/SitterFlow/ScheduleView/CreateNewWalkingView.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+
+/// A view providing tools for selecting date, time, and recurrence options for
+/// creating walkings in a sitter's schedule.
+struct CreateNewWalkingView: View {
+    var body: some View {
+        VStack(spacing: 30) {
+            VStack {
+                Text(createWalkingTitle)
+                    .font(.headline)
```

Comment:
Regarding our AppStyleGuide we define 3 font sizes we can use. Please choose from them